src/middlewared/middlewared/plugins/cluster_linux/ctdb_shared_volume.py

```diff
@@ -19,31 +18,23 @@ class Config:
         namespace = 'ctdb.shared.volume'
         private = True
 
-    async def exists_and_started(self):
-
-        exists = started = False
-
+    async def validate(self):
         filters = [('id', '=', CTDB_VOL_NAME)]
-        vol = await self.middleware.call('gluster.volume.query', filters)
-        if vol:
-            if vol[0]['type'] != 'REPLICATE':
-                raise CallError(
-                    f'A volume with the name "{CTDB_VOL_NAME}" already exists '
-                    'but is not a REPLICATE type volume. Please delete or rename '
-                    'this volume and try again.'
-                )
-            elif vol[0]['replica'] < 3 or vol[0]['num_bricks'] < 3:
-                raise CallError(
-                    f'A volume with the name "{CTDB_VOL_NAME}" already exists '
-                    'but is configured in a way that could cause split-brain. '
-                    'Please delete or rename this volume and try again.'
-                )
-            elif vol[0]['status'] != 'Started':
-                exists = True
-            else:
-                exists = started = True
-
-        return exists, started
+        for i in await self.middleware.call('gluster.volume.query', filters):
+            err_msg = f'A volume named "{CTDB_VOL_NAME}" already exists '
+            if i['type'] != 'REPLICATE':
+                err_msg += '''but is not a "REPLICATE" type volume. \
+                    Please delete or rename this volume and try again.'''
```

Comment:
I think this would add that many spaces to the err msg?

Proper would be:
```
err_msg += ("but is not a "REPLICATE" type volume. "
    "Please delete or rename this volume and try again.")

Reply:
It didn't in my testing but I can use the tuple approach no problem.